src/checkbox/ux-checkbox.ts

```diff
@@ -1,4 +1,5 @@
 import { customElement, bindable, ViewResources, View, processAttributes } from 'aurelia-templating';
+import { computedFrom } from 'aurelia-framework';
```

Comment:
This is really a minor thing, but we should put computedFrom into the 'aurelia-binding' import rather than making a new one referencing 'aurelia-framework' to be consistent.